src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/SaveFileListener.java

```diff
@@ -41,19 +48,28 @@ public void actionPerformed(ActionEvent evt) {
             );
         }
 
-        try {
-            BufferedWriter writer = new BufferedWriter(new FileWriter(window.getFile()));
-            writer.write(window.getEditorTab().getText());
-            writer.close();
-        } catch (IOException e) {
-            LOG.severe(e.getMessage());
-            JOptionPane.showMessageDialog(
-                    window,
-                    "Could not save file: "+e.getMessage(),
-                    "Error",
-                    JOptionPane.ERROR_MESSAGE
-            );
+        else{
```

Comment:
This looks like it should be *Save As...* instead of *Save*. *Save As* traditionally asks you where you would like to save the file, while *Save* would save over your current file.

I would put this into a `SaveAsListener` and add a `JMenu` item for `Save As...`, so this listener can continue to be used for the standard *Save* function